tests/virtualization/yast_virtualization.pm

```diff
@@ -8,15 +8,21 @@ sub run() {
     wait_idle;
     send_key "alt-f10";
     become_root;
-    script_run "yast2 virtualization";
-    wait_idle;
-    assert_screen "virt-sle12sp1-gnome_yast_virtualization", 50;
-    # select everything
-    send_key "alt-x", 10;    # XEN Server
-    send_key "alt-e", 10;    # Xen tools
-    send_key "alt-k", 10;    # KVM Server
-    send_key "alt-v", 10;    # KVM tools
-    send_key "alt-l", 10;    # libvirt-lxc
+    script_run("/sbin/yast2 virtualization; echo yast2-virtualization-done-\$? > /dev/$serialdev", 0);
+    assert_screen "virt-sle-gnome_yast_virtualization";
+    if (check_var("FLAVOR", "Desktop-DVD")) {
```

Comment:
the same argument as above applies here: why would the workflow be different between server and desktop?